modal/image.py

```diff
@@ -582,6 +582,15 @@ def conda(python_version: str = "3.9") -> "_Image":
         return _Image(
             dockerfile_commands=dockerfile_commands,
             context_files={"/modal_requirements.txt": requirements_path},
+        ).dockerfile_commands(
```

Comment:
Not sure if this makes sense, but I appended this as an sub-image rather than include it in the base to make rebuilds quicker. Let me know if you think I should rather move it to the base image instead